src/NServiceBus.AcceptanceTests/Core/OpenTelemetry/Metrics/TestingMetricListener.cs

```diff
@@ -36,7 +36,18 @@ public TestingMetricListener(string sourceName)
             ReportedMeters.AddOrUpdate(instrument.Name, measurement, (_, val) => val + measurement);
             Tags.AddOrUpdate(instrument.Name, _ => tags, (_, _) => tags);
         });
-        meterListener.Start();
+        meterListener.SetMeasurementEventCallback((Instrument instrument,
+            double measurement,
```

Comment:
The measurement is a duration right? Can we make that more explicit? This is a nit, so feel free to ignore.

Reply:
I'm not following, what should be more explicit?